app/models/subscriber.rb

```diff
@@ -0,0 +1,6 @@
+class Subscriber < ApplicationRecord
+  self.table_name = "subscriber"
+
+  validates :webhook_url, presence: true, uniqueness: { scope: :subscriber_type }
+  validates :subscriber_type, presence: true
+end
```

Comment:
I still think it might be worth having a failed count stored on this record (i..e how many attempts to send have failed in a row) this means if/when the unsibscribe fails we can use this as a secondary way to unsubscribe (potentially)

Reply:
Good shout. I've added it, but the auto-delete-above-threshold is only configured to run on dev to avoid screwing us in production.